### PR TITLE
SDK-Core Subgraph Tests

### PR DESCRIPTION
**Pull request by 0xdavinchee**
* Use MATIC for v1/release
* Use GOERLI for feature and dev

### CODE REVIEW

File: packages/sdk-core/previous-versions-testing/runQueryTests.ts [hunk 1]
```diff
@@ -1,9 +1,7 @@
 import { Query } from "@superfluid-finance/sdk-core/src";
+import { chainIdToResolverDataMap } from "@superfluid-finance/sdk-core/src/constants";
```
Comment (kasparkallas) [inline packages/sdk-core/previous-versions-testing/runQueryTests.ts:2]: I don't see this import being used.

File: packages/sdk-core/previous-versions-testing/queryTests.ts [hunk 2]
```diff
@@ -1,5 +1,20 @@
 import { Query } from "@superfluid-finance/sdk-core/src";
 import { expect } from "chai";
+import {
+    ETH_GOERLI_CHAIN_ID,
+    MATIC_CHAIN_ID,
```
Comment (kasparkallas) [inline packages/sdk-core/previous-versions-testing/queryTests.ts:5]: These imports look ugly cosmetically. :smile: Would have preferred an enum or something.

**Review by kasparkallas — APPROVED** [review]

File: packages/sdk-core/previous-versions-testing/runQueryTests.ts (hunk 1 above)
Comment (0xdavinchee) [reply packages/sdk-core/previous-versions-testing/runQueryTests.ts:2]: it's used on line 13 by `getSubgraphEndpoint`

File: packages/sdk-core/previous-versions-testing/queryTests.ts (hunk 2 above)
Comment (0xdavinchee) [reply packages/sdk-core/previous-versions-testing/queryTests.ts:5]: true, can be something to refactor for SDK-core in the future, but there were some strange type issues w/ enums in the past that were kinda annoying

**Pull request merged**